### ISSUE TITLE
local-config:generate - session save option chooser

### ISSUE THREAD

**Issue by amenk** [netz98/n98-magerun#4]
The parameter "session save" should give several options to choose from.

**Comment by robbieaverill** [comment]
I know this is old, but happy to help. Do you suggest those options being `db` and `files`? My understanding is that centralized session storage like Redis would use its own configuration tag, and the `session_save` tag would become redundant whatever it contained.

**Comment by amenk** [comment]
db and files would be fine for the beginning. I do not understand your comment about redis - do you mean that more config is needed for redis?

**Comment by robbieaverill** [comment]
@amenk yeah - at least for older versions of Magento. `db` and `files` seems a sensible place to start!
